hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/FormatUtils.java

```diff
@@ -120,9 +123,12 @@ private static Object getVal(IndexedRecord record, int pos) {
   public static HoodieMergedLogRecordScanner logScanner(
       MergeOnReadInputSplit split,
       Schema logSchema,
-      Configuration config,
-      boolean withOperationField) {
```

Comment:
We can pass in `writeConfig` instead of flink configuration instead.

Reply:
Oops, sorry, i checked the code and find it is better to construct the `writeConfig` in `#logScanner`, you original code that passes in Flink Configuration is better :), sorry for the mistake. You can just construct the writeConfig inside `#logScanner` is only this method uses the write config.

Reply:
yes u are right 👍 
hadoopConf does not support serialization